apps/gobstopper_api/lib/gobstopper.api/auth.ex

```diff
@@ -25,4 +25,12 @@ defmodule Gobstopper.API.Auth do
     """
     @spec verify(token) :: uuid | nil
     def verify(token), do: GenServer.call(@service, { :verify, token })
+
+    @doc """
+        Refresh an identity's session.
```

Comment:
Should be 2 spaces following the `@doc` segment. As 4 will make it a code block in markdown.